cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,89 @@
 /// <reference types='cypress' />
 
+let user;
+
+function addUser(count) {
+  for (let i = 0; i < count; i++) {
+    cy.findById('addNewRecordButton').click();
+    cy.findById('firstName').type(user.firstName);
+    cy.findById('lastName').type(user.lastName);
+    cy.findById('userEmail').type(user.email);
+    cy.findById('age').type(user.age);
+    cy.findById('salary').type(user.salary);
+    cy.findById('department').type(user.department);
+
+    cy.findById('submit').click();
+    cy.get('.rt-td').should('contain', user.email);
+  }
+}
+
 describe('Web Tables page', () => {
-  it('', () => {
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+  });
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('Should add a new worker', () => {
+    addUser(1);
+  });
+
+  it('Should delete a user', () => {
+    cy.findById('delete-record-3').click();
+    cy.contains('.rt-td', 'kierra@example.com').should('not.exist');
+  });
+
+  it('Should delete all users', () => {
+    cy.findById('delete-record-1').click();
```

Comment:
The test assumes that a user with the id 'delete-record-1' exists. Ensure that this user is present before running this test, or dynamically create the user in a 'beforeEach' hook.